src/types/api.ts

```diff
@@ -452,6 +454,24 @@ export interface paths extends PathRegistry {
       }
     }
   }
+  '/v1/auth/nonce': {
+    get: operations['get_auth_nonce']
+    parameters: {
+      path: null
+      credentials: RequestCredentials
+    }
+  }
+  '/v1/auth/verify': {
+    post: operations['verify_auth']
+    parameters: {
+      path: null
+      credentials: RequestCredentials
```

Comment:
This can only be `include`, right? The endpoint doesn't make sense w/ a different value.

```suggestion
      credentials: 'include'
```

Reply:
Makes sense!